C/include/c4Database.h

```diff
@@ -45,11 +45,6 @@ extern "C" {
         kC4DB_NonObservable = 0x40, ///< Disable c4DatabaseObserver
     };
 
-    /** Document versioning system (also determines database storage schema) */
-    typedef C4_ENUM(uint32_t, C4DocumentVersioning) {
-        kC4RevisionTrees,           ///< Revision trees
-    };
```

Comment:
Does this mean we are committed to either not doing version vectors, or switching to them without the option to stay on rev trees if and when we do?

Reply:
The latter, probably.